hedy.py

```diff
@@ -1444,6 +1445,7 @@ def transpile_inner(input_string, level):
 
 def execute(input_string, level):
     python = transpile(input_string, level)
+    print(python)
```

Comment:
I think you  might have added this for debugging? Can be removed in the final PR.

Reply:
Yes, sorry, this can be removd! I actually removed most of them and forgot thid one.

Reply:
No problem, this has happened to me often too! Will you do it or do you want me to?

Reply:
Ill do it, Im having conection problems but Ill upload it.